website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -109,6 +109,18 @@ The following attributes are exported:
 
 * `fqdn` - The FQDN of the Azure Kubernetes Managed Cluster.
 
+* `kube_config.0.client_key` - Base64 encoded client key used to authenticate to the Kubernetes cluster
+
+* `kube_config.0.client_certificate` - Base64 encoded client certificate used to authenticate to the Kubernetes cluster
+
+* `kube_config.0.cluster_ca_certificate` - Base64 encoded cluster CA certificate used to authenticate to the Kubernetes cluster
+
+* `kube_config.0.host` - The Kubernetes cluster server host
+
+* `kube_config.0.username` - The username used to authenticate to the Kubernetes cluster
+
+* `kube_config.0.password` - The password or token used to authenticate to the Kubernetes cluster
```

Comment:
could we update this to match the other documentation? In general we'd show these like so:

```
* `kube_config` - a `kube_config` block as defined below:

---

* `client_key` - The base64-encoded Client Key used to authenticate to the Kubernetes Cluster

# ...
```

it may be worth adding an output to the example above to demonstrate how to access them?